src/git.ts

```diff
@@ -184,7 +184,7 @@ export async function deploy(action: ActionInterface): Promise<void> {
           ? action.commitMessage
           : `Deploying to ${action.branch} from ${action.baseBranch}`
       } ${
-        process.env.GITHUB_SHA ? `- ${process.env.GITHUB_SHA}` : ''
+        process.env.GITHUB_SHA ? `@ ${process.env.GITHUB_SHA}` : ''
```

Comment:
Wanted to make this suggestion. IMHO `@` seems like a better fit here given how Git tools refer to `$something` _at_ a certain revision.